addons/mail/static/src/models/follower/follower.js

```diff
@@ -257,6 +258,9 @@ function factory(dependencies) {
                 'partnerName',
             ],
         }),
+        displayName: attr({
```

Comment:
I think fields should be ordered alphabetically in JS models (except resID), should probably be defined between channelName and followedThread (cc @seb-odoo )

Reply:
Corrected, thanks for the hint!